### PR TITLE
Endpoint to login verified 'worker' user

### LINKED ISSUE #4: Worker login
Worker will be able to login using their credentials. 

- This will consist of implementing integration with Reputation Oracle POST /auth/signin endpoint. 
- The backend will mirror the request parameters and response structure from reputation oracle as it will be a pure proxy endpoint.

### PR DESCRIPTION
**Pull request by MWBlocky**
## Description

This Pull Request adds the login endpoint for type worker user.

## Summary of changes

- Added signin endpoint.
- Environment variables for e2e tests.
- Unit & e2e tests.

## Related issues
Fixes #4

### CODE REVIEW

File: packages/apps/human-app/server/src/common/config/environment-config.service.ts [hunk 1]
```diff
@@ -14,10 +14,18 @@ export class EnvironmentConfigService {
   get reputationOracleUrl(): string {
     return this.configService.get<string>('REPUTATION_ORACLE_URL', '');
   }
+  get e2eTestingEmailAddress(): string {
+    return this.configService.get<string>('E2E_TESTING_EMAIL_ADDRESS', '');
+  }
+  get e2eTestingPassword(): string {
+    return this.configService.get<string>('E2E_TESTING_PASSWORD', '');
+  }
```
Comment (BartoszSolkaBD) [inline packages/apps/human-app/server/src/common/config/environment-config.service.ts:22]: test related stuff shouldn't be in the production code. It should be in test directory or at least a separate file for test config

**Review by BartoszSolkaBD — CHANGES_REQUESTED** [review]

File: packages/apps/human-app/server/src/common/config/environment-config.service.ts (hunk 1 above)
Comment (MWBlocky) [reply packages/apps/human-app/server/src/common/config/environment-config.service.ts]: Solved in 4865ae76

File: packages/apps/human-app/server/src/modules/user-worker/spec/worker.controller.spec.ts [hunk 2]
```diff
@@ -54,4 +55,19 @@ describe('WorkerController', () => {
       expect(workerService.signupWorker).toHaveBeenCalledWith(expectedCommand);
     });
   });
+
+  describe('signin', () => {
+    it('should worker service signin method with proper fields set', async () => {
```
Comment (macnablocky) [inline packages/apps/human-app/server/src/modules/user-worker/spec/worker.controller.spec.ts:60]: Please clarify name of the test

Comment (MWBlocky) [reply packages/apps/human-app/server/src/modules/user-worker/spec/worker.controller.spec.ts]: Solved in: a63253f0

**Review by BartoszSolkaBD — APPROVED** [review]

**Pull request merged**